suites/nautilus/upgrades/upgrade_3.x_lvm_to_4.x_lvm.yaml

```diff
@@ -0,0 +1,99 @@
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    abort-on-fail: True
+- test:
+    name: ceph ansible install rhcs 3 stable
+    module: test_ansible.py
+    config:
+      use_cdn: True
+      build: '3.0'
+      ansi_config:
+        ceph_test: True
+        ceph_origin: repository
+        ceph_repository: rhcs
+        ceph_repository_type: cdn
+        ceph_rhcs_version: 3
+        ceph_stable_release: luminous
+        osd_scenario: lvm
+        osd_auto_discovery: False
+        journal_size: 1024
+        copy_admin_key: True
+        ceph_conf_overrides:
+          global:
+            osd_pool_default_pg_num: 128
+            osd_default_pool_size: 2
+            osd_pool_default_pgp_num: 128
+            mon_max_pg_per_osd: 4096
+            mon_allow_pool_delete: True
+          client:
+            rgw crypt require ssl: false
+            rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+              testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+    desc: test cluster setup using ceph-ansible
+    abort-on-fail: True
+- test:
+    name: ceph ansible
+    module: test_ansible_upgrade.py
```

Comment:
polarion ID missing, check if there is one for normal upgrade or any new polarion ID included as part of rhcs4.x plan

Reply:
@Ameenasuhani could you add the polariton id here as we discussed. let's try to close this as fast as possible. its been pending for some time now.